### PR TITLE
Latest Python 2 and 3 Documentation

### PR DESCRIPTION
**Pull request by cjmayo**
New https SRC_URI and HOMEPAGE.

For 2.7.12 I did remove pkg_postrm() as compared to 2.7.10

### CODE REVIEW

**Comment by mgorny** [comment]
Do you want to play a bit with modernizing those ebuilds? I could give you a few short tasks if you're interested in helping out ;-).

**Comment by cjmayo** [comment]
Let's go for it.

Do you actually want separate commits for 2 & 3?

**Comment by mgorny** [comment]
That would be probably nice but I don't mind either way.

File: dev-python/python-docs/python-docs-2.7.12.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (mgorny) [inline dev-python/python-docs/python-docs-2.7.12.ebuild]: So let's go for EAPI=6.

File: dev-python/python-docs/python-docs-2.7.12.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://www.python.org/doc/"
+SRC_URI="https://docs.python.org/2/archives/python-${PV}-docs-html.tar.bz2"
+
+LICENSE="PSF-2"
+SLOT="2.7"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~amd64-fbsd ~sparc-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE=""
+
+DEPEND=""
+RDEPEND=""
```
Comment (mgorny) [inline dev-python/python-docs/python-docs-2.7.12.ebuild]: You can drop the two.

File: dev-python/python-docs/python-docs-2.7.12.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+S="${WORKDIR}/python-${PV}-docs-html"
+
+src_install() {
+	dohtml -A xml -A inv -r ./
```
Comment (mgorny) [inline dev-python/python-docs/python-docs-2.7.12.ebuild]: Would be nice if you gave it a run with:

    PORTAGE_DOHTML_WARN_ON_SKIPPED_FILES=yes

in make.conf and seen if Portage complains about some files being skipped. If yes, paste them and we'll consider if they should be skipped or installed.

Then you'd need to replace `dohtml` with `docinto` + `dodoc`.

File: dev-python/python-docs/python-docs-2.7.12.ebuild [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	echo "PYTHONDOCS_${SLOT//./_}=\"${EPREFIX}/usr/share/doc/${PF}/html/library\"" > "60python-docs-${SLOT}"
```
Comment (mgorny) [inline dev-python/python-docs/python-docs-2.7.12.ebuild]: || die ;-)

**Comment by cjmayo** [comment]
EAPI, DEPENDs and die done.

PORTAGE_DOHTML_WARN_ON_SKIPPED_FILES reported the hidden file .buildinfo and the  _sources directory and all its contents.

I experimented with HTML_DOCS in the latest versions. No problem with replacing with docinto, dohtml. More change needed anyway because this does install _sources (and .buildinfo). Add a ```rm -r _sources || die``` before?

**Comment by mgorny** [comment]
Yes, that sounds about right ;-).

**Comment by cjmayo** [comment]
Hopefully that's it!

**Comment by Arfrever** [comment]
https://bugs.python.org/issue25170 is not yet fixed.
https://docs.python.org/*/archives/ is mentioned as something which will have continuously rebuilt versions

**Comment by cjmayo** [comment]
OK. Dropped the 3.5.2 ebuild, pointed 2.7.12 at https://www.python.org/ftp/python/doc/ and regenerated the Manifest.

**Comment by monsieurp** [comment]
@gentoo/python

**Pull request closed without merging**